websocket/src/websocket.cpp

```diff
@@ -709,6 +712,13 @@ static dmExtension::Result OnUpdate(dmExtension::Params* params)
 #if defined(__EMSCRIPTEN__)
             conn->m_SSLSocket = dmSSLSocket::INVALID_SOCKET_HANDLE;
 
+            if (conn->m_Protocol) {
+                EM_ASM({
```

Comment:
So this changes the sub-protocol for all active web sockets? Could that become a problem? Or is the sub-protocol only set when connecting?

Reply:
It might be an issue.
I couldn't really find how the `defold-websocket` solved this issue?

Another option would be to set it at the app initialization phase.